heudiconv/tests/test_dicoms.py

```diff
@@ -85,3 +87,51 @@ def test_group_dicoms_into_seqinfos(tmpdir):
     assert type(seqinfo) is OrderedDict
     assert len(seqinfo) == len(dcmfiles)
     assert [s.series_description for s in seqinfo] == ['AAHead_Scout_32ch-head-coil', 'PhoenixZIPReport']
+
+
+
+def test_get_datetime_from_dcm():
+    import datetime
+    typical_dcm = dcm.dcmread(op.join(TESTS_DATA_PATH, 'phantom.dcm'), stop_before_pixels=True)
+    XA30_enhanced_dcm = dcm.dcmread(op.join(TESTS_DATA_PATH, 'MRI_102TD_PHA_S.MR.Chen_Matthews_1.3.1.2022.11.16.15.50.20.357.31204541.dcm'), stop_before_pixels=True)
```

Comment:
I don't know what is our widest line , but this might be aiming to become a winner!  Could we keep them within e.g. 120 chars?